s3-autodelete.sh

```diff
@@ -6,11 +6,13 @@
 set -e
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+date_today=`date`
```

Comment:
I'm confused how this variable is used? It looks like a plain string?

Reply:
It gets today's date and used when calculating the max date when deleting backups.

Reply:
Ah I see, just ran the command on my local and confirmed that.